owtf/db/database.py

```diff
@@ -41,11 +46,15 @@ def wrapper(self, *args, **kwargs):
 
 
 def get_db_engine():
-    engine = create_engine(
+    try:
+        engine = create_engine(
             "postgresql+psycopg2://{}:{}@{}/{}".format(DATABASE_USER, DATABASE_PASS, DATABASE_IP, DATABASE_NAME),
-                pool_recycle=120)
-    Base.metadata.create_all(engine)
-    return engine
+            pool_recycle=120)
```

Comment:
Catch the exception like in https://github.com/owtf/owtf/blob/master/owtf/db/database.py#L147.

Reply:
Why aren't we using DB_PORT while creating the engine? I think we should use it since it will allow user to explicitly define the port on which postgres is running using `settings.py`.

Reply:
Agreed. Change this to use value from `settings.py`.